### PR TITLE
doc: releases update the luminous, hammer, jewel release dates

### PR DESCRIPTION
**Pull request by theanalyst**
Updated the table with released luminous and hammer info.

### CODE REVIEW

File: doc/releases.rst [hunk 1]
```diff
@@ -23,7 +23,9 @@ Timeline
 |                |Development|`Dumpling`_|`Emperor`_ |`Firefly`_ |`Giant`_   |`Hammer`_  |`Infernalis`_ |`Jewel`_   |`Kraken`   |
 |                |Testing    |LTS        |Stable     |LTS        |Stable     |LTS        |Stable        |LTS        |Stable     |
 +----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
-| January   2017 |           |           |           |           |           |`0.94.10`_ |              |           |`11.2.0`_  |
+| January   2017 |           |           |           |           |           |           |              |           |`11.2.0`_  |
++----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
```
Comment (dachary) [inline doc/releases.rst]: reverse ?

Comment (smithfarm) [inline doc/releases.rst]: yeah, the months are in reverse order

Comment (theanalyst) [inline doc/releases.rst]: Ah ok, got it now, fixing

Comment (smithfarm) [inline doc/releases.rst]: also, the links need to point to something (see under the table)

**Comment by theanalyst** [comment]
@dachary updated

File: doc/releases.rst [hunk 2]
```diff
@@ -23,9 +23,12 @@ Timeline
...
 |                |Testing    |LTS        |Stable     |LTS        |Stable     |LTS        |Stable        |LTS        |Stable     |
 +----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
-| January   2017 |           |           |           |           |           |`0.94.10`_ |              |           |`11.2.0`_  |
+| February  2017 | `12.0.0`_ |           |           |           |           |`0.94.10`_ |              |           |           |
++----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
+| January   2017 |           |           |           |           |           |           |              |           |`11.2.0`_  |
 +----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
 | December  2016 |           |           |           |           |           |           |              |`10.2.4`_  |           |
+|                |           |           |           |           |           |           |              |`10.2.5`_  |           |
```
Comment (smithfarm) [inline doc/releases.rst]: @theanalyst Could you put 10.2.5 first and on a separate table row, for consistency? (see below)

Comment (smithfarm) [inline doc/releases.rst]: See, e.g., April 2015, October 2014 etc.

Comment (theanalyst) [inline doc/releases.rst]: ack, will do

File: doc/releases.rst [hunk 3]
```diff
@@ -23,15 +23,20 @@ Timeline
...
 +----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
-| January   2017 |           |           |           |           |           |`0.94.10`_ |              |           |`11.2.0`_  |
+| February  2017 | `12.0.0`_ |           |           |           |           |`0.94.10`_ |              |           |           |
++----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
+| January   2017 |           |           |           |           |           |           |              |           |`11.2.0`_  |
 +----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
 | December  2016 |           |           |           |           |           |           |              |`10.2.4`_  |           |
++                +-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
+|                |           |           |           |           |           |           |              |`10.2.5`_  |           |
```
Comment (smithfarm) [inline doc/releases.rst]: Reverse the order so 10.2.5 is above 10.2.4

Comment (smithfarm) [inline doc/releases.rst]: The `+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+` line doesn't look long enough?

**Comment by smithfarm** [comment]
Jenkins retest this please

**Comment by theanalyst** [comment]
@dachary is this okay?

**Pull request merged**